### PR TITLE
Fix the cr/lf format in the Auto OBM setting Readme file

### PR DESCRIPTION
**Pull request by nortonluo**
The original Readme is in Windows text format and have CR/LF ending. That led to web page unable to render correctly. Re-format the readme into Unix.

### CODE REVIEW

File: obm-autoset/README.md [hunk 1]
```diff
@@ -1,47 +1,47 @@
-#RackHD IPMI OBM Setting Auto Set Tool
-Tool to apply OBM setting for each compute node discovered by RackHD. Save the trouble to manually apply OBM setting node by node.
-
-##Usage
-Downlaod this folder.
-Make sure you have python 2 or python 3 installed
-cd to the cloned directory.
-Change the environment setting in obm_setting.json file.
-python ApplyOBMSetting.py -ora 192.168.129.96
```
Comment (anhou) [inline obm-autoset/README.md]: please make this command line as a code block

File: obm-autoset/README.md [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
\ No newline at end of file
+# RackHD IPMI OBM Setting Auto Set Tool
+Tool to apply OBM setting for each compute node discovered by RackHD. Save the trouble to manually apply OBM setting node by node.
+
+## Usage
+Downlaod this folder.
+Make sure you have python 2 or python 3 installed
+cd to the cloned directory.
+Change the environment setting in obm_setting.json file.
+python ApplyOBMSetting.py -ora 192.168.129.96
```
Comment (anhou) [inline obm-autoset/README.md]: please make this command line as a code block

File: obm-autoset/README.md [hunk 3]
```diff
@@ -1,47 +1,47 @@
...
+
+### -user
+Override the usernames in OBM setting file, default is "None"
+
+### -password
+Override the password in OBM setting file, default is "None"
+
+
+
+### -v {0,2,4,6,9}
```
Comment (anhou) [inline obm-autoset/README.md]: maybe '-v' option is not the same style with other options like 'user' 'password' ....

File: obm-autoset/README.md [hunk 4]
```diff
@@ -1,47 +1,47 @@
...
+Make sure you have python 2 or python 3 installed
+cd to the cloned directory.
+Change the environment setting in obm_setting.json file.
+python ApplyOBMSetting.py -ora 192.168.129.96
+
+## Theory of Operation
+The script will check the RMM/BMC address information in the catalog of ora. And then put OBM setting to ora by RackHD 2.0 API. The compute node should be discovered by RackHD ora first.
+
+## Supported parameters
+### -ora
```
Comment (anhou) [inline obm-autoset/README.md]: 'ora' is outdated statement for RackHD, you could use 'host' or similar words

File: obm-autoset/README.md [hunk 5]
```diff
@@ -1,47 +1,47 @@
...
+### -nodeid {NODEID}
+Specify one compute node to apply OBM setting instead of apply OBM setting to all Nodes. default:all")
+
+### -auth {on/off}
+Specify if authentication is on/off , default:off
+
+
+## obm_setting.json Config File
+This file contains the credentials required to access the ora/bmcs. It includes:
+"ora": RackHD ora ssh credential.
```
Comment (anhou) [inline obm-autoset/README.md]: no line break for this parts

It includes: "ora": RackHD ora ssh credential. "bmc": bmc/rmm credential list. "api": RackHD api login credential "ports":http/https tcp port number.

please click 'view' button at the right-top to have a review.

File: obm-autoset/README.md (hunk 4 above)
Comment (nortonluo) [inline obm-autoset/README.md]: No. this is only a readme format fix for the old tool. If this is changed, the script need modify

**Pull request merged**